css/main.css

```diff
@@ -27,7 +27,34 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+.mobile-menu {
+  display: none;
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  background-color: #3c3a39;
+  z-index: 100;
+  flex-direction: column;
+  padding: 20px 20px 0 20px;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.overflow-hidden {
+  overflow: hidden;
+}
+
+.mobile-menu-close {
+  display: flex;
+  justify-content: flex-end;
+  font-size: 30px;
+  color: red;
```

Comment:
You forgot to change the color of `x` button as the last reviewer mentioned so please change it and try to stick as much as possible to the given [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). 

![](https://user-images.githubusercontent.com/34578631/157398967-d2b9489c-ad80-4058-982f-64ddcdf69540.png) 
![image](https://user-images.githubusercontent.com/68796232/157731941-97b11a0e-0cee-42f8-b835-12db9b7f0d73.png)